testsuite/integration/clustering/pom.xml

```diff
@@ -1537,6 +1654,44 @@
                                     </configurations>
                                 </configuration>
                             </execution>
+                            <!-- Provision 'datasources-web-server' with 'singleton-local' -->
+                            <execution>
+                                <id>singleton-local-provisioning</id>
+                                <goals>
+                                    <goal>provision</goal>
+                                </goals>
+                                <phase>compile</phase>
+                                <configuration>
+                                    <install-dir>${project.build.directory}/wildfly-singleton-local</install-dir>
+                                    <record-state>false</record-state>
+                                    <log-time>${galleon.log.time}</log-time>
+                                    <offline>true</offline>
+                                    <plugin-options>
+                                        <jboss-maven-dist/>
+                                        <jboss-fork-embedded>${galleon.fork.embedded}</jboss-fork-embedded>
+                                        <optional-packages>passive+</optional-packages>
+                                    </plugin-options>
```

Comment:
N.B. This will impact https://github.com/wildfly/wildfly/pull/15730

Reply:
Right, thanks, that's painfully clear. :-)